app/src/main/res/values/strings.xml

```diff
@@ -11,4 +11,14 @@
     <string name="warning_close">Close</string>
     <string name="warning_more">More info</string>
     <string name="warning_text">For this app to work, you need to enable remote debugging on the phone using following adb command:\n\nadb tcpip 5555</string>
+    <string name="hint_filter_by_tag">Filter by tag</string>
+    <string name="menu_delete_all">Delete all</string>
+    <string name="menu_filter">Filter</string>
```

Comment:
Please sort string so they are grouped by areas. For example, it makes sense for all `menu_` to be next to each other so it's easier to add/modify/remove anything.